jac-uk/admin
# Feature/2552 Updates to character task/report in line with changes

## What's included?
Closes #2552 

- Update the "Agency report" on the application view.

Related PR: [digital-platform: Feature/admin 2552 Update character task/report in line with changes #1215](https://github.com/jac-uk/digital-platform/pull/1215)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

[Example application](https://jac-admin-develop--pr2576-feature-2552-updates-mutr5irw.web.app/exercise/Biyjd07Xz2usL9yXjtjV/applications/applied/application/DmtSTqTUDD2d0aihvOQ2#agency)

1. Go to the Agency report tab of an application.
2. Check if the data under the HMRC section displays correctly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/user-attachments/assets/f92894b3-fcef-4f22-a342-0e18878ee88f

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
